### DISCUSSION TITLE
Restore problem

### DISCUSSION THREAD

**Discussion by Ararat95** [community/community#33204] [Actions]
I used git clean -f -d in home directory and deleted all the files i have, can anyone help me, i've tried everything but nothing in trash and cant restore anything... pls help me!

**Comment by jfullstackdev** [comment]
it's a local copy ? you did not push it to GitHub ?

**Comment by Ararat95** [comment]
Yes that all was local files, including all directories which are not going
to be pushed, no i didnt push , i recover 1 of them, but others are lost ,
they are not even in trash!

пт, 16 сент. 2022 г., 16:41 jdevstatic ***@***.***>:

**Comment by airtower-luna** [comment]
If you don't have a backup I'm afraid your data is most likely gone forever. :slightly_frowning_face: 

You could try to find a data rescue service in your area (basically they'd search your disk for blocks that look like they might belong to files), but that's expensive and the result uncertain.
